### DISCUSSION TITLE
Tooltips + Popovers in Quarto Dashboard Cards

### DISCUSSION THREAD

**Discussion by mrworthington** [quarto-dev/quarto-cli#9385] [Q&A]
### Description

Hi! 

I've recently been building a lot of Quarto Dashboards that combine R Data wrangling + OJS interactivity. The Dashboard system is great, but I'm wondering if there's any plans (or maybe just non-existent documentation) on how to leverage bslib's [card tooltip + popover features](https://rstudio.github.io/bslib/articles/tooltips-popovers/index.html)?

Right now, I'm resorting to more hacky/custom approaches to getting the user the kind of functionality provided by tooltips + popovers. Wanted to see if this was on the radar of the Quarto team? 

Thanks in advance!

**Comment by mcanouil** [comment]
There are quite many things to be added/improved, unfortunately no timeline can be provided.
Note that `bslib` is very R driven while Quarto is more agnostic, so the experience should be the same when using Jupyter Notebook, Python code, and R.
Also note, that there is some integration via `bslib`: https://quarto.org/docs/dashboards/data-display.html#shiny
